src/Accounts/Accounts/ChangeLog.md

```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 
+## Version 2.10.1
+* Fixed returning duplicate Ids for one subscription while using `Get-AzSubscription` with parameter `SubscriptionName`. [#19427]
```

Comment:
We don't need to update the version manually. Just add changelog content under `Upcoming Release` section is fine.